recipes/cymove/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "cymove" %}
+{% set version = "1.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ozars/cymove/archive/v{{ version }}.tar.gz
```

Comment:
I can't use the sidst from PyPI due to files missing but I've opened an issue about it: https://github.com/ozars/cymove/issues/1